src/constants/locConstants.ts

```diff
@@ -920,6 +920,7 @@ export class ContainerDeployment {
         });
     };
     public static configureLinuxContainers = l10n.t("Configure Linux containers");
+    public static configureRosetta = l10n.t("Configure Rosetta Virtualization");
```

Comment:
We might want to remove `Virtualization` from the text as Rosetta is a translation layer not a virtualization layer. (Ofc under the hood it still requires Apple Virtualization Framework but that's unrelated to Rosetta itself).
https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment